--- Pillow-7.0.0/src/PIL/__init__.py.orig	2020-01-16 21:24:20.388014146 +0100
+++ Pillow-7.0.0/src/PIL/__init__.py	2020-01-17 05:31:07.749784296 +0100
@@ -23,6 +23,13 @@
 del _version
 
 
+def _get_optional_plugins():
+    try:
+        import SpiderImagePlugin
+        return ['SpiderImagePlugin']
+    except:
+        return []
+
 _plugins = [
     "BlpImagePlugin",
     "BmpImagePlugin",
@@ -60,7 +67,6 @@
     "PpmImagePlugin",
     "PsdImagePlugin",
     "SgiImagePlugin",
-    "SpiderImagePlugin",
     "SunImagePlugin",
     "TgaImagePlugin",
     "TiffImagePlugin",
@@ -69,7 +75,7 @@
     "XbmImagePlugin",
     "XpmImagePlugin",
     "XVThumbImagePlugin",
-]
+] + _get_optional_plugins()
 
 
 class UnidentifiedImageError(IOError):
