--- Pillow-8.1.0/src/PIL/__init__.py.orig	2021-01-02 20:54:29.000000000 +0100
+++ Pillow-8.1.0/src/PIL/__init__.py	2021-02-06 16:05:42.826876555 +0100
@@ -82,6 +82,13 @@
 del _version
 
 
+def _get_optional_plugins():
+    try:
+        import SpiderImagePlugin
+        return ['SpiderImagePlugin']
+    except:
+        return []
+
 _plugins = [
     "BlpImagePlugin",
     "BmpImagePlugin",
@@ -119,7 +126,6 @@
     "PpmImagePlugin",
     "PsdImagePlugin",
     "SgiImagePlugin",
-    "SpiderImagePlugin",
     "SunImagePlugin",
     "TgaImagePlugin",
     "TiffImagePlugin",
@@ -128,7 +134,7 @@
     "XbmImagePlugin",
     "XpmImagePlugin",
     "XVThumbImagePlugin",
-]
+] + _get_optional_plugins()
 
 
 class UnidentifiedImageError(OSError):
