diff -urNp -x '*.orig' Pillow-8.1.0.org/src/PIL/__init__.py Pillow-8.1.0/src/PIL/__init__.py
--- Pillow-8.1.0.org/src/PIL/__init__.py	2021-01-02 20:54:29.000000000 +0100
+++ Pillow-8.1.0/src/PIL/__init__.py	2021-02-28 11:37:59.771120362 +0100
@@ -82,6 +82,13 @@ else:
 del _version
 
 
+def _get_optional_plugins():
+    try:
+        import SpiderImagePlugin
+        return ['SpiderImagePlugin']
+    except:
+        return []
+
 _plugins = [
     "BlpImagePlugin",
     "BmpImagePlugin",
@@ -119,7 +126,6 @@ _plugins = [
     "PpmImagePlugin",
     "PsdImagePlugin",
     "SgiImagePlugin",
-    "SpiderImagePlugin",
     "SunImagePlugin",
     "TgaImagePlugin",
     "TiffImagePlugin",
@@ -128,7 +134,7 @@ _plugins = [
     "XbmImagePlugin",
     "XpmImagePlugin",
     "XVThumbImagePlugin",
-]
+] + _get_optional_plugins()
 
 
 class UnidentifiedImageError(OSError):
