diff -urNp -x '*.orig' Pillow-8.1.0.org/setup.py Pillow-8.1.0/setup.py
--- Pillow-8.1.0.org/setup.py	2021-01-02 20:54:29.000000000 +0100
+++ Pillow-8.1.0/setup.py	2021-02-28 11:38:43.717892626 +0100
@@ -423,6 +423,8 @@ class pil_build_ext(build_ext):
 
         #
         # add platform directories
+        _add_directory(library_dirs, "/libx32")
+        _add_directory(library_dirs, "/usr/libx32")
 
         if self.disable_platform_guessing:
             pass
